arroyo-console/src/routes/pipelines/JobDetail.tsx

```diff
@@ -422,6 +454,13 @@ function OperatorDetail({
           Events TX
           <TimeSeriesGraph data={msgSentData} timeWindowMs={5 * 60 * 1000} />
         </Box>
+
+        {backpressureData?.some(m => m.length) && (
```

Comment:
I'm not sure it makes sense to have this as a graph -- generally this doesn't vary over time much (at not on the 5 minute timescale of these graphs).

Reply:
Fair, maybe we should just make it a badge to indicate the current backpressure status.

Reply:
I removed the graph and added a badge instead.